src/python/pants/bin/local_pants_runner.py

```diff
@@ -106,7 +106,13 @@ def _maybe_init_graph_session(graph_session, options_bootstrapper,build_config,
 
       v2_ui = options.for_global_scope().v2_ui
       zipkin_trace_v2 = options.for_scope('reporting').zipkin_trace_v2
-      graph_session = graph_scheduler_helper.new_session(zipkin_trace_v2, RunTracker.global_instance().run_id, v2_ui)
+      #TODO(gregorys) This should_report_workunits flag must be set to True for
```

Comment:
Rather than a username, TODOs that are worth landing with should have github issue numbers associated with them.